extensions/amp-ad/0.1/amp-ad-ui.js

```diff
@@ -65,12 +66,29 @@ export class AmpAdUIHandler {
    * Order: try collapse -> apply provided fallback -> apply default fallback
    */
   applyNoContentUI() {
-    if (getAdContainer(this.element_) == 'AMP-STICKY-AD') {
+    if (getAdContainer(this.element_) === 'AMP-STICKY-AD') {
       // Special case: force collapse sticky-ad if no content.
       this.baseInstance_./*OK*/collapse();
       return;
     }
 
+    if (getAdContainer(this.element_) === 'AMP-FX-FLYING-CARPET') {
+
+      // Special case: force collapse ad if only child of a flying carpet.
+      const flyingCarpetElements =
```

Comment:
Is it easier just to check `this.element_.parentElement`

Reply:
So flying carpet actually has two nested container elements, thus `this.element_.parentElement` won't work 😢 

<img width="573" alt="screen shot 2018-12-03 at 4 52 29 pm" src="https://user-images.githubusercontent.com/1448289/49410976-f0260000-f71b-11e8-9316-434ba8ea73d5.png">